src/components/Nav.js

```diff
@@ -57,13 +59,30 @@ function RightSide({user}) {
   );
 }
 
-function Header({user, handleLogOut, handleLogIn}) {
+function Header({user, handleLogOut, handleLogIn, isAdmin}) {
+  const [adminBar, setAdminBar] = useState(localStorage.getItem("adminBar") === "true");
```

Comment:
I don't think the `=== "true"` is needed. Since the `adminBar &&` is happening below if it's null or undefined it will work.

Reply:
@bdougie **localstorage** always returns as a **string** so I'm **casting** in this way!

`adminBar &&` handles only boolean right!

Reply:
got it. You are correct, and this makes sense.